src/test/php/predicate/ArraySubsetTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare(strict_types=1);
+/**
+ * This file is part of bovigo\assert.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace bovigo\assert\predicate;
+use PHPUnit\Framework\TestCase;
+
+use function bovigo\assert\assertFalse;
+use function bovigo\assert\assertTrue;
+/**
+ * Tests for bovigo\assert\predicate\arraySubset.
+ *
+ * @group  predicate
```

Comment:
Please add a `@since`tag here as well. A `@group issue_22` would be appreciated.